### ISSUE TITLE
Properly handle WebCite links

### ISSUE THREAD

**Issue by razasyedh** [zhaofengli/refill#8]
I've noticed that you handle Wayback Machine links differently than normal links and parse archive information from them. Would you consider handling [WebCite](http://www.webcitation.org/) links as well? They are [also used extensively to combat link rot](https://en.wikipedia.org/wiki/Wikipedia:Link_rot#Web_archive_services). Currently, it seems that the service just reads any archived URL's title as "WebCite query result" and the work as "webcitation.org". You can find an example of this [here](https://en.wikipedia.org/w/index.php?title=Naum_Levin&diff=653066491&oldid=629448799). Note that the URLs [come in two formats](https://en.wikipedia.org/wiki/Wikipedia:Using_WebCite#Use_within_Wikipedia).

**Comment by zhaofengli** [comment]
WebCite pages are pretty problematic as there is likely no useful information in the URLs (most links only include a 9-digit ID), and the returned pages include the actual archived content with `iframe`s, which means additional HTTP requests are needed to fetch the archived pages themselves and the WebCite metadata. Maybe we can find a clean way to do this inside a `MetadataParser`. Thanks for the report!

**Comment by razasyedh** [comment]
Yeah the effort might not be worth it if the 9-digit form is more common. And bare WebCite URL's might be few (3,600 total Google hits vs archive.org's 35,000). If anything it'd be worth adding the domain to the list of sites that reFill can't handle.

You're welcome. Thanks for maintaining this tool!

**Comment by zhaofengli** [comment]
Okay, I've [blacklisted](https://github.com/zhaofengli/refill-labsconf/commit/8bcebd724d2806df90027701a82de83e3eb319a5) webcitation.org while we are looking for a solution.

**Comment by owcz** [comment]
If you already have a scrape running on the page, the original URL is displayed in WebCite archives at /html/body/table/tbody/tr[1]/td[2]/a[1] (table[@class="topframe"])

**Comment by Artoria2e5** [comment]
Consider checking out the workings of [Wayback Medic](https://en.wikipedia.org/wiki/User:Green_Cardamom/WaybackMedic_2.1).

**Comment by greencardamom** [comment]
I'm the author of WaybackMedic and can answer questions (User:Green Cardamom). The archive date can be obtained from the 9-digit ID .. it is a base62 number converted from 10-digit Unix time. For a decoder see my userpage under "Software". WebCite also has an API.

**Comment by nemobis** [comment]
Webcitation.org is once again winding down and (on the English Wikipedia) there's already wayback medic to handle the past URLs, so in my opinion this task can be declined. We don't need every tool to do everything. On non-English Wikipedias, WebCite is used even less; on all Wikimedia projects I checked, it's two orders of magnitude less common than wayback machine.

**Comment by CurbSafeCharmer** [comment]
@greencardamom do you agree with @nemobis that this issue can be closed?
